src/main/java/dev/andrylat/vomelianchuk/card_number_operations/CardValidationException.java

```diff
@@ -0,0 +1,24 @@
+package dev.andrylat.vomelianchuk.card_number_operations;
```

Comment:
You put everything in the same package. But you have absolutely different pieces of functionality and you can split them and put in the different packages.

Reply:
Fixed here: 7a87509